services/command-router-infinispan/pom.xml

```diff
@@ -0,0 +1,92 @@
+<!--
+    Copyright (c) 2021, 2022 Contributors to the Eclipse Foundation
+
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.hono</groupId>
+    <artifactId>hono-services-parent</artifactId>
+    <version>2.6.0-SNAPSHOT</version>
+    <relativePath>../parent</relativePath>
+  </parent>
+  <artifactId>hono-service-command-router-infinispan</artifactId>
+  <name>Hono Infinispan Based Command Router</name>
+  <description>A Quarkus based implementation of Hono's Command Router API that is using Infinispan for storing data.</description>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>hono-service-command-router-base</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>client-device-connection-infinispan</artifactId>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>io.fabric8</groupId>
+          <artifactId>docker-maven-plugin</artifactId>
+          <configuration>
+            <images>
+              <image>
```

Comment:
we will need to make sure to also tag the created image with the original name, i.e. `hono-service-command-router`, in addition to the one derived from the Maven artifact name.
This is to provide backwards compatibility with existing Helm charts etc.

Reply:
Great catch. I was going to mention this and ask for your opinion but forgot to do that preemptively.

So I'd personally prefer if we could keep the `command-router-infinispan` module name, I think that's much clearer than having `command-router-base`, `command-router` and `command-router-redis`.

So my hope was that we could somehow add it to the CI that once the `command-router-infinispan` container had been created and tagged, it would simply also be uploaded as `command-router` using the same tags.

Is this something that could be done you think? Perhaps by modifying the `push_hono_images.sh` script somehow? I didn't want to do that as I'm not sure exactly how the CI process works here.

Or would you maybe prefer that the module is just named `command-router`?

Reply:
> So my hope was that we could somehow add it to the CI that once the command-router-infinispan container had been created and tagged, it would simply also be uploaded as command-router using the same tags.

yes, I think that should work. Sadly, the docker-maven-plugin doesn't seem to support to tag the created image with multiple names, it can only add additional tags ...

Reply:
I believe I've addressed all other comments and resolved them.

I'm not sure what the best way forward for this would be though and I'm not seeing exactly how the official releases of Hono are published to the registry. Do you do that manually using the `push_hono*_images.sh` scripts?

Essentially, the only thing that is needed would be to run the following:
```
docker tag "eclipse/hono-service-command-router-infinispan:${TAG}" "eclipse/hono-service-command-router:${TAG}"
docker tag "eclipse/hono-service-command-router-infinispan-native:${TAG}" "eclipse/hono-service-command-router-native:${TAG}"
docker push "eclipse/hono-service-command-router:${TAG}"
docker push "eclipse/hono-service-command-router-native:${TAG}"
```

Should I try to figure out how to add that to the `push_hono_images.sh` script?

Reply:
> Do you do that manually using the push_hono*_images.sh scripts?

Yes, that is the current mechanism.

Reply:
I've updated the `push_hono_images.sh` script such that it now tags the command-server image with its legacy name and includes that in the push to the docker registry.